modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/OrganizationResourceTest.java

```diff
@@ -268,18 +268,22 @@ protected String[] getAdditionalAssertFieldNames() {
 			testDeleteAccountByExternalReferenceCodeOrganization_addOrganization()
 		throws Exception {
 
-		return organizationResource.putOrganizationByExternalReferenceCode(
-			StringUtil.toLowerCase(RandomTestUtil.randomString()),
-			randomOrganization());
+		Organization organization =
+			organizationResource.putOrganizationByExternalReferenceCode(
+				_accountEntry.getExternalReferenceCode(), randomOrganization());
```

Comment:
Why are you using the ERC of the account entry to assign it to the organization?

Reply:
My fault, I saw it in other tests but it make sense, I'm going to correct it